### ISSUE TITLE
Call to undefined method Dtc\QueueBundle\ODM\JobManager::pruneErroneousJobs()

### ISSUE THREAD

**Issue by kralos** [mmucklo/DtcQueueBundle#72]
| Q             | A             |
| ------------- | ------------- |
| Version       | 4.5.3         |
| Persistance   | ODM           |

config:

```yaml
dtc_queue:
    manager:
        job: odm
    timings:
        record: true
```

`console -v dtc:queue:prune error` causes `Call to undefined method Dtc\QueueBundle\ODM\JobManager::pruneErroneousJobs()`

**Comment by mmucklo** [comment]
@kralos thanks for reporting, I'll take a look

**Comment by kralos** [comment]
FYI, not sure if you would rather steer ODM users away from prune and instead point the in the direction of TTL indexes: https://docs.mongodb.com/manual/core/index-ttl/

**Comment by kralos** [comment]
```
kralos@maia:~/src/jobs/vendor/mmucklo/queue-bundle$ git log -1
commit 868eb806b5935a3b5b99e9bc73e70e8c74fe2d8d (HEAD, origin/master, origin/HEAD)
Author: Matthew J. Mucklo <mmucklo@gmail.com>
Date:   Mon Jun 11 21:42:13 2018 -0700

    fixes for issues #72 and #73
kralos@maia:~/src/jobs/vendor/mmucklo/queue-bundle$ cd ../../..
kralos@maia:~/src/jobs$ bin/console -v dtc:queue:prune old_job_timings --older '1d'
0 Archived Job Timing(s) pruned
kralos@maia:~/src/jobs$ bin/console -v dtc:queue:prune stalled --older '1d'
0 Stalled Job(s) pruned
kralos@maia:~/src/jobs$ bin/console -v dtc:queue:prune stalled_runs --older '1d'
0 Stalled Job(s) pruned
kralos@maia:~/src/jobs$ bin/console -v dtc:queue:prune exception --older '1d'
0 Job(s) with status 'exception' pruned
kralos@maia:~/src/jobs$ bin/console -v dtc:queue:prune expired --older '1d'
0 Expired Job(s) pruned
kralos@maia:~/src/jobs$ bin/console -v dtc:queue:prune old --older '1d'
17 Archived Job(s) pruned
kralos@maia:~/src/jobs$ bin/console -v dtc:queue:prune old_runs --older '1d'
32 Archived Run(s) pruned
kralos@maia:~/src/jobs$ bin/console -v dtc:queue:prune old_job_timings --older '1d'
0 Archived Job Timing(s) pruned
kralos@maia:~/src/jobs$ 
```

**Comment by mmucklo** [comment]
Suggested heeded (README.md updated) and issue fixed as of 4.6.0
